### PR TITLE
Implement CloneUnauthorizedElementSpec

### PR DESCRIPTION
**Pull request by sdelamo**
See https://github.com/MetadataConsulting/ModelCataloguePlugin/issues/1193

### CODE REVIEW

**Comment by sdelamo** [comment]
@Vendetta30  Could you review the test? It fails for me. Moreover, please review the instructions of the test.

**Comment by sdelamo** [comment]
Instead of creating new Geb classes : 

`DataModelPermissionGrantPage` and `DataModelPermissionListPage`

we should remove those and use the already existing: 

`DataModelAclPermissionsShowPage` `DataModelAclPermissionsPage`

File: ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/DataModelAclPermissionsShowPage.groovy [hunk 1]
```diff
@@ -18,22 +18,22 @@ class DataModelAclPermissionsShowPage extends Page {
...
+        nav { $('#topmenu', 0).module(NavModule) }
     }
 
     int count() {
         rows.size()
     }
 
     void setUsername(String value) {
-        usernameSelect = value
+        usernameSelect(value).click()
```
Comment (vijayshukla30) [inline ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/DataModelAclPermissionsShowPage.groovy:32]: An issue with selecting a user from the drop-down.

File: ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/DataModelAclPermissionsShowPage.groovy [hunk 2]
```diff
@@ -18,22 +18,22 @@ class DataModelAclPermissionsShowPage extends Page {
...
     }
 
     void setPermission(String value) {
-        permissionSelect = value
+        permissionSelect(value).click()
```
Comment (vijayshukla30) [inline ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/DataModelAclPermissionsShowPage.groovy:36]: An issue with selecting roles from the drop-down.

**Comment by vijayshukla30** [comment]
@sdelamo 
We have implemented this, as you suggested. But, there was one issue which we faced while selecting user/roles from the drop-down. So instead of assigning a value to the drop-down, we performed click action on it.
I attached the screenshot of the error, please let me know your suggestions.
screenshot from 2018-04-24 10-50-25

**Pull request merged**